packages/cron/src/bin/dagcargo-sizes.js

```diff
@@ -4,14 +4,18 @@ import { updateDagSizes } from '../jobs/dagcargo.js'
 import { envConfig } from '../lib/env.js'
 import { getPg } from '../lib/utils.js'
 
-const oneMonthAgo = () => new Date().setMonth(new Date().getMonth() - 1)
+/**
+ *
+ * @param {number} days
+ */
+const xDaysAgo = (days) => new Date().setDate(new Date().getDate() - days)
```

Comment:
Not sure we had a whole month here?

I've updated to take just a week into consideration.
It could actually be even shorter given the cron runs every 38 minutes.

Reply:
Thanks